cicd/pipeline.jenkins

```diff
@@ -0,0 +1,18 @@
+def repo = params.get("REPO") 
```

Comment:
As with the other jenkins files, a description would be good. Do we want to have some sort of correlation between 'sh' env vars and these jenkins vars? Should they be consistently named?